src/client/editor/nodes/FloorPlanNode.js

```diff
@@ -282,6 +279,19 @@ export default class FloorPlanNode extends EditorNodeMixin(FloorPlan) {
     return this;
   }
 
+  async loadNavMesh(src) {
```

Comment:
Does this actually relate to the media stuff at all or did it just slip in?

Reply:
This branch is based off of the `feature/farspark-model` branch. I'm surprised this is still showing up in the diff. https://github.com/MozillaReality/Spoke/pull/386/files#diff-3a1b4a86fe9a09ae9d0623b70ac64336R282